### PR TITLE
Remove unused dart:async import

### PR DESCRIPTION
**Pull request by MichaelRFairhurst**
This change does require depending on Dart >=2.1, if that's acceptable.

### CODE REVIEW

**Comment by MichaelRFairhurst** [comment]
ping @mogol

**Comment by MichaelRFairhurst** [comment]
ping again @mogol

**Comment by mogol** [comment]
Hey @MichaelRFairhurst 
thanks for MRs, looking into

ps Need to check notifications, I missed the notifications again

File: pubspec.yaml [hunk 1]
```diff
@@ -18,4 +18,4 @@ dependencies:
     sdk: flutter
 
 environment:
-    sdk: ">=1.19.0 <3.0.0"
+    sdk: ">=2.1.0 <3.0.0"
```
Comment (mogol) [inline pubspec.yaml:21]: Why do we need this?

Comment (MichaelRFairhurst) [reply pubspec.yaml:21]: As of Dart 2.1, Future and Stream have been exported from dart:core. I'm adding tooling to catch these imports, and fixing them where I can.

Dart 2.1 was launched Nov 15, 2018, however, if you still want to support pre-Dart 2.1, it's totally acceptable to close this PR :)

**Comment by mogol** [comment]
Thanks :+1:

**Pull request merged**